app/models/spree/credit_card_decorator.rb

```diff
@@ -3,6 +3,9 @@
   # Should be able to remove once we reach Spree v2.2.0
   # Commit: https://github.com/spree/spree/commit/5a4d690ebc64b264bf12904a70187e7a8735ef3f
   # See also: https://github.com/spree/spree_gateway/issues/111
+
+  belongs_to :payment_method
```

Comment:
Need to shift this line up or down, it's currently splitting the comment [above] from the method to which it applies [below].